cbits/odbc.c

```diff
@@ -101,7 +106,8 @@ EnvAndDbc *odbc_AllocEnvAndDbc(){
         EnvAndDbc *envAndDbc = malloc(sizeof *envAndDbc);
         envAndDbc->env = env;
         envAndDbc->dbc = dbc;
-        envAndDbc->error = malloc(MAXBUFLEN);
+        envAndDbc->error = malloc(SQL_MAX_MESSAGE_LENGTH);
+        envAndDbc->sqlState = malloc(6);
```

Comment:
```suggestion
        # SQLSTATE is a five-character code, see https://docs.microsoft.com/en-us/sql/odbc/reference/appendixes/appendix-a-odbc-error-codes?view=sql-server-ver15
        envAndDbc->sqlState = malloc(6);
```

Reply:
Ouch, I think I was writing a bash script be before adding this suggestion comments should start with `//` in C :)